Tests/Tests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Tests.swift
+//  Tests
+//
+//  Created by Igor Ranieri on 08.01.18.
+//
+
+import XCTest
+
+class Tests: XCTestCase {
```

Comment:
I think you can get rid of this boilerplate

Reply:
Xcode being funny, by removing the file from the project but not deleting it from the FS.